README.md

```diff
@@ -16,8 +16,53 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Prefer CSS animations over .animate()](#prefer-css-animations-over-animate)
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Prefer CSS animations over [.animate()](http://api.jquery.com/animate/)
+
+### Why?
+Most of the time the same can be accomplished with toggling classes and using CSS transition and/or keyframes.
+
+### How?
+``` javascript
+
+// Bad
+function animateElement() {
+  $myElement.animate({
+    	opacity: 0
+    },
+    1000)
+    .animate({
+    	opacity: 1
+    },
+    1000,
+    function() {
+    	animateElement();
+    });
+}
+animateElement();
+```
+
+```javascript
+// Good
+$myElement.addClass('is-animating');
+```
+
+``` css
+/* vendor prefix might be required */
+.is-animating {
+    animation: fade 2s infinite;
+}
+
+@keyframes fade {
```

Comment:
I would say a keyframe animation is a more complex example of a CSS alternative than something we could solve with CSS transitions. Should we add another example to "How?" before this one with a CSS transition?